src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java

```diff
@@ -59,6 +60,7 @@
     private final MetricRegistry metricRegistry;
     private final ClosedConnectionsCrutch closedConnectionsCrutch;
 
+    @Autowired
     public EventStreamController(final EventTypeRepository eventTypeRepository, final TopicRepository topicRepository,
```

Comment:
I think it's fine. You already made it explicit when a `@DB` is required, so it's all fine 👍